taxonomies/ingredients_analysis.txt

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Could you also add this new taxonomy to `refresh_taxonomies.js`, so that Travis CI can rebuild it?

Reply:
Sure.